src/search-schema/data/definitions/enums/assay_types.yaml

```diff
@@ -171,23 +171,41 @@ salmon_rnaseq_bulk:
     primary: false
     vitessce-hints: []
 
-SNAREseq:
-    description: SNARE-seq
+SNARE2-ATACseq:
+    description: snATACseq (SNARE-seq2)
+    alt-names: [SNAREseq]
+    primary: true
+    vitessce-hints: []
+
+SNARE2-RNAseq:
+    description: snRNAseq (SNARE-seq2)
     alt-names: []
     primary: true
     vitessce-hints: []
 
 sc_atac_seq_snare_lab:
 #SNAREseq_lab:
-    description: scATAC-seq (SNARE-seq) [Lab Processed]
+    description: snATAC-seq (SNARE-seq2) [Lab Processed]
 #    alt-names: [sc_atac_seq_snare_lab]
     alt-names: []
     primary: false
     vitessce-hints: []
 
+sc_rna_seq_snare_lab:
+    description: snRNA-seq (SNARE-seq2) [Lab Processed]
+    alt-names: []
+    primary: false
+    vitessce-hints: []
+
+salmon_rnaseq_snareseq:
+    description: snRNA-seq (SNARE-seq2) [Salmon]
+    alt-names: []
+    primary: false
+    vitessce-hints: ['is_sc', 'rna']
+
 sc_atac_seq_snare:
 #SNAREseq_snapATAC:
-    description: scATAC-seq (SNARE-seq) [SnapATAC]
+    description: snATAC-seq (SNARE-seq2) [SnapATAC]
```

Comment:
```suggestion
    description: snATAC-seq (SNARE2-seq) [SnapATAC]
```

Reply:
We will keep `SNARE-seq2` and change the keys to match that pattern.